source/include/core_json.h

```diff
@@ -106,7 +109,6 @@ JSONStatus_t JSON_Validate( const char * buf,
  * @param[in] max  size of the buffer.
```

Comment:
Why isn't `bufSize` or `bufferSize` used as the parameter name instead of `max`? The use of `max` suggested **max depth** to be instead of buffer size when I saw the name..

Reply:
Well, it made sense to me and the previous reviewers.  It's part of almost every function.

Reply:
It wouldn't hurt to update just the public-facing functions' parameters to make it easier for the customer to understand. I am fine with keeping `max` as-is within the private functions in the implementation

Reply:
Changing the name in the declaration and not in the implementation throws a MISRA 8.3 error.  I'd rather have consistency in core_json.c.